samples/test/ssmlAddresses.test.js

```diff
@@ -0,0 +1,56 @@
+/**
+ * Copyright 2019 Google LLC
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict';
+
+const fs = require('fs');
+const {assert} = require('chai');
+
+const ssmlAddresses = require('./../ssmlAddresses');
+
+//const cmd = 'node ssmlAddresses.js';
+const inputFile = 'resources/example.txt';
+const outputFile = 'test-example.mp3';
+
+describe('ssmlAddresses', () => {
+  it('input text tagged with SSML', () => {
+    let expected_ssml = '';
+    try {
```

Comment:
I think you can remove the try/catch blocks - whatever errors would happen, the traceback would be printed as part of test failure